docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -69,57 +69,45 @@ The following guidelines describe the required structure for your collection:
 
 Controller environment
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Collections MUST support all eligible controller Python versions in the controller environment, unless required libraries do not support these Python versions. The :ref:`Steering Committee <steering_responsibilities>` can grant other exceptions on a case-by-case basis. 
+* Collections MUST support all eligible controller Python versions in the controller environment, unless required libraries do not support these Python versions. The :ref:`Steering Committee <steering_responsibilities>` can grant other exceptions on a case-by-case basis. 
+
+  * controller environment: the plugins/modules always run in the same environment (Python interpreter, venv, host, and so on) as ansible-core itself.
+  * eligible controller Python version: a Python version that is supported on the controller side by at least one ansible-core version that the collection supports. The eligible versions can be determined from the :ref:`ansible_core_support_matrix` and from the ``requires_ansible`` value in ``meta/runtime.yml`` in the collection.
 
-The collection MUST document all eligible controller Python versions that are not supported in the controller environment. See :ref:`coll_python_docs_req` for details.
+* The collection MUST document all eligible controller Python versions that are **not** supported in the controller environment. See :ref:`coll_python_docs_req` for details.
 
 Other environment
 ~~~~~~~~~~~~~~~~~
 
-Collections MUST support all eligible controller Python versions in the other environment, unless required libraries do not support these Python versions. The :ref:`Steering Committee <steering_responsibilities>` can grant other exceptions on a case-by-case basis. 
+* Collections MUST support all eligible controller Python versions in the other environment, unless required libraries do not support these Python versions. The :ref:`Steering Committee <steering_responsibilities>` can grant other exceptions on a case-by-case basis. 
 
-Collections SHOULD support all eligible target Python versions in the other environment.
+  * other environment: the plugins/modules run not in a controller environment.
+  * eligible target Python version: a Python version that is supported on the target side by at least one ansible-core version that the collection supports. The eligible versions can be determined from the :ref:`ansible_core_support_matrix` and from the ``requires_ansible`` value in ``meta/runtime.yml`` in the collection.
 
-The collection MUST document all eligible target Python versions that are not supported in the other environment. See :ref:`coll_python_docs_req` for details.
-
-.. note::
-
-    Note that dropping support for a Python version for an existing module/plugin is a breaking change, and thus requires a major release. A collection MUST announce dropping support for Python versions in their changelog, if possible in advance (for example, in previous versions before support is dropped).
```

Comment:
This note has been simply removed. Since it requires a MUST that changes the requirements.

Reply:
@felixfontein I believe it actually doesn't change the requirements as this statement just elaborates on SemVer which already covers such cases.
So it just removes duplication.
Moreover, if we request exactly this case as MUST, it's imo impossible/not reasonable to control it, e.g. if we find out that they didn't announce it will we reject the collection inclusion request/kick out it from the package?
Main argument to remove this: they must just follow SemVer in general.

Reply:
> @felixfontein I believe it actually doesn't change the requirements as this statement just elaborates on SemVer which already covers such cases.

Since there has been disagreement multiple times on IRC whether dropping Python support is a breaking change, I think this is really important.

>  Moreover, if we request exactly this case as MUST, it's imo impossible/not reasonable to control it, e.g. if we find out that they didn't announce it will we reject the collection inclusion request/kick out it from the package?

Same as any other violation: we point the problem out to the collection and let them fix it.

Reply:
i was thinking that it'll be hard to identify this even at the review stage compared to items like "releases are tagged in the repository". but to point this out wouldn't hurt, i'll put it back, thanks

Reply:
@felixfontein i've put it back in a bit different format, PTAL